### ISSUE TITLE
Mac FlutterFlow does not accept AppleId for login

### ISSUE THREAD

**Issue by Chasbo** [FlutterFlow/flutterflow-issues#2348]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I joined FlutterFlow online using AppleId for my login. That works for online use. However, the Mac app (which I would like to be able to use) has no ability to login in with AppleId.

Bluntly, these seems like a perverse oversight.

### Expected Behavior

I expect to be able to use the same login everywhere I wish to develop FlutterFlow.

### Steps to Reproduce

Simply view the Mac FlutterFlow login dialog.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

NA

### Context

Cannot develop using native app

### Visual documentation

NA

### Additional Info

NA

### Environment

```markdown
Mac app Version 4.1.27 (4.1.27)
```

**Comment by hariprasadms** [comment]
Hi @Chasbo, Thanks for submitting the issue. The current Mac version of FlutterFlow does not have apple login capability and this is as per current design. You should consider available authentication methods to use the FlutterFlow. However, I can mark this issue as feature and enhancement request. Thanks

**Comment by Chasbo** [comment]
I am sorry that the issue was apparently not understood. I think that I did not make myself clear.

I do not want to add AppleID verification to a completely Flutter app built with FlutterFlow. That is NOT the issue.

Notice this screenshot of the FlutterFlow account setup page from the FlutterFlow site; it offers five options for account ID including Apple.

￼


However, when I decided to try the Mac app, it only allows three ways to login to the account. Here is a screenshot from the very latest Mac app just downloaded. Somehow, Apple and Microsoft have not made the cut. This is, at a minimum, a pain. It is likely that I will need to delete my account and start a new one. I am sure you can agree that this is not the most welcoming of new user experiences.


￼

Seems to me that this is a FlutterFlow tool bug and one that really does need fixing. You could either get Apple/Microsoft off the account creation page or you could make it work in the app. Either would do.

**Comment by leighajarett** [comment]
Hi there, the issue is understood. This is something we're working on supporting. In the meantime, you can reset your password to use standard username / password authentication

**Comment by ABIV** [comment]
> Hi there, the issue is understood. This is something we're working on supporting. In the meantime, you can reset your password to use standard username / password authentication

I don't think this works (at leas on the web) because the Apple login by passes a password. Trying to reset password never sends an email, and you can't change the account email address without a password.

**Comment by leighajarett** [comment]
Oh no, that's not good. I would have expected that you received an email to reset password. We will take a closer look

**Comment by rzambroni** [comment]
Hi @ABIV & @Chasbo, thanks a lot for the report. We understand the inconvenience, the team is working hard to have all options available for every platform. However, the workaround @leighajarett mentioned is valid. I just tested it myself. 

Please follow the reset password flow:
- Go to the FlutterFlow Native app and click on the "Forgot Password?" button
- Enter your email (the same email associated with your Apple Sign-in account you used to create your account on the web app)
- Go to your inbox, find flutterflow's reset password email and select a password
- You will be able to log in with your email and password on all platforms and also use Apple-sign-in on the webapp.

Thanks.

**Comment by ABIV** [comment]
@rzambroni I don't think this works if you used hide my email. Is the reset email coming from the right domain? I am not getting the reset email from web or desktop.

**Comment by rzambroni** [comment]
@ABIV, yep, that is correct. If you are not sharing your email address FlutterFlow will not be able to send you the restore password email. If that is the case there is no other option rather than waiting until apple sign in is added to the platform you want to use. 
I believe you can check on any IOS device if you are sharing your email with the app by going to Settings -> AppleID -> Password and Security -> Apple Sign-in -> finding FF on the list & Checking data shared.

**Comment by leighajarett** [comment]
Don't have a timeline at the moment, sorry. Not that many users sign in with Apple so it's a bit lower priority. 

Sorry again 🙁 I know that's super annoying

**Comment by jovani-io** [comment]
I have the same issue, using signin with apple using <myapplegeneratedusername>@privaterelay.appleid.com, I tried password reset, update my email which ask for a password, which do not work, no email is sent. I was locked up on using the web app, I want to use the Mac flutterflow app but no continue with apple option.

**Comment by d002054** [comment]
We urgently need the same feature as we are currently migrating our existing app to Flutterflow.

In our current app, we use Auth0 as our login provider, which includes the option for users to sign in with their Apple ID in the WebApp. Many of our users with Apple subscriptions use the @privaterelay.appleid.com email address and access the WebApp on their desktop.

Without this feature, our users would only be able to use the iOS version of our app and not the desktop version.

Thank you for your outstanding support.

**Comment by leighajarett** [comment]
Users can login to FlutterFlow using Apple, this feature is specifically for FlutterFlow users to sign in to the FlutterFlow mac application using their Apple ID
